### DISCUSSION TITLE
MJPEG Stream for specific camera in the world

### DISCUSSION THREAD

**Discussion by OlliePugh** [cyberbotics/webots#5543] [Q&A]
I am wondering if there is a bulit-in solution to view a specific camera in the world via the web MJPEG stream? Hoping to use Webot to train a ML model for a real world environment where the IRL camera outputs an MJPEG stream, so would be super easy if I could keep that same design for the simulation.

**Comment by omichel** [comment]
You will have to implement this by yourself, as there is not built-in solution for this.

**Reply by OlliePugh** [reply]
Alrighty! Thank you for your quick response!
